tests/native_tests/language.hy

```diff
@@ -1037,20 +993,6 @@
   (assert (is (xor 0 False) False)))
 
 
-(defn test-if-return-branching []
```

Comment:
I don't know. I can't get a version of Hy that old to run in order to try to reproduce the bug without `let`.